### PR TITLE
fix bug #1003 #1005 #997

### PR DESCRIPTION
**Pull request by digger-yu**
fix bug #1003 #1005 #997
Including link error, spelling error and garbled code

### CODE REVIEW

File: doc/trex_vm_manual.asciidoc [hunk 1]
```diff
@@ -382,11 +382,11 @@ zmq publisher at: tcp://*:4500
 <1> Output trimmed.
 
 [NOTE]
-See http://trex-tgn.cisco.com/trex/doc/trex_book.pdf[TRex full manual] for a complete understading of the tool features and options.
+See link:https://trex-tgn.cisco.com/trex/doc/trex_manual.html[TRex full manual] for a complete understading of the tool features and options.
```
Comment (hhaim) [inline doc/trex_vm_manual.asciidoc:385]: This better be relative 

```
link:trex_manual.html[TRex full manual]
```
please convert all the other links to relative

Comment (digger-yu) [reply doc/trex_vm_manual.asciidoc]: Sorry for waiting so long.

**Comment by digger-yu** [comment]
@hhaim please check

**Comment by hhaim** [comment]
rebased

**Pull request closed without merging**